src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private Integer number;
+
+    public Ball(Color color, Integer number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return this.color.toString() + this.number.toString();
```

Comment:
Consider adding a separator between `color` and `number` in the `toString` method for better readability. For example, you could use a space or a hyphen.